### PR TITLE
[enterprise-4.14] OSDOCS-6677: add RN entry

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/OSDOCS-8064

Link to docs preview: https://65889--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-new-cert-process

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -361,6 +361,25 @@ With this release, you can now provision an {product-title} cluster with Tang-en
 
 For more information, see xref:../installing/install_config/installing-customizing.adoc#installation-special-config-encryption-threshold_installing-customizing[Configuring an encryption threshold] and xref:../installing/install_config/installing-customizing.adoc#installation-special-config-storage-procedure_installing-customizing[Configuring disk encryption and mirroring].
 
+[id="ocp-4-14-new-cert-process"]
+==== Certificates are now handled by the Machine Config Daemon
+
+In previous {product-title} versions, the MCO read and handled certificates directly from machine configuration files. This led to rotation issues and created unwanted situations, like when certificates were stuck behind a paused machine config pool.
+
+With this release, certificates are no longer templated from bootstrap into machine configuration files. Instead, they are put directly onto disk using controller configs and handled by the Machine Config Daemon (MCD). The certs are then visible using the Controller Config.
```
Comment (cdoern) [inline release_notes/ocp-4-14-release-notes.adoc:369]:
```suggestion
With this release, certificates are no longer templated from bootstrap into machine configuration files. Instead, they are put directly into the ignition object, written onto disk using the controller config, and handled by the Machine Config Daemon (MCD) during regular cluster operation. The certs are then visible using the Controller Config.
```

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -361,6 +361,25 @@ With this release, you can now provision an {product-title} cluster with Tang-en
...
+
+The MCD handles the following certificates:
+
+* `/etc/kubernetes/kubelet-ca.crt`
+* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
+* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`
+
+This means that certificates are not bound by the machine config pool status and are more timely in their rotation.
+
+Any CAs stored in machine configuration files have been removed, and the templated files found during cluster installation no longer exist.
```
Comment (cdoern) [inline release_notes/ocp-4-14-release-notes.adoc:379]:
```suggestion
The above listed CAs stored in machine configuration files have been removed, and the templated files found during cluster installation no longer exist.
```

**Review by cdoern — APPROVED** [review]
Looks good, slight changes.

**Comment by jldohmann** [comment]
@rioliu-rh could you PTAL for QE ack? This is the RN entry for updates to how certain certificates are handled in the MCO. Thank you!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -361,6 +361,25 @@ With this release, you can now provision an {product-title} cluster with Tang-en
 
 For more information, see xref:../installing/install_config/installing-customizing.adoc#installation-special-config-encryption-threshold_installing-customizing[Configuring an encryption threshold] and xref:../installing/install_config/installing-customizing.adoc#installation-special-config-storage-procedure_installing-customizing[Configuring disk encryption and mirroring].
 
+[id="ocp-4-14-new-cert-process"]
+==== Certificates are now handled by the Machine Config Daemon
+
+In previous {product-title} versions, the MCO read and handled certificates directly from machine configuration files. This led to rotation issues and created unwanted situations, like when certificates were stuck behind a paused machine config pool.
+
+With this release, certificates are no longer templated from bootstrap into machine configuration files. Instead, they are put directly into the Ignition object, written onto disk using the controller config, and handled by the Machine Config Daemon (MCD) during regular cluster operation. The certs are then visible using the Controller Config.
```
Comment (skrthomas) [inline release_notes/ocp-4-14-release-notes.adoc:369]:
```suggestion
With this release, certificates are no longer templated from bootstrap into machine configuration files. Instead, they are put directly into the Ignition object, written onto a disk using the controller config, and handled by the Machine Config Daemon (MCD) during regular cluster operation. The certs are then visible using the Controller Config.
```

Comment (skrthomas) [reply release_notes/ocp-4-14-release-notes.adoc:369]: I think an a/the is needed before "disk". Also, I was wondering about Controller Config needing to be capitalized? In the docs, I see config controller, lowercase. I'm unfamiliar with this, so I just wanted to ask. 

Also is Ignition an API object that needs backticks?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 4]
```diff
@@ -361,6 +361,25 @@ With this release, you can now provision an {product-title} cluster with Tang-en
...
+
+The MCD handles the following certificates:
+
+* `/etc/kubernetes/kubelet-ca.crt`
+* `/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem`
+* `/etc/pki/ca-trust/source/anchors/openshift-config-user-ca-bundle.crt`
+
+This means that certificates are not bound by the machine config pool status and are more timely in their rotation.
+
+The above listed CAs stored in machine configuration files have been removed, and the templated files found during cluster installation no longer exist.
```
Comment (skrthomas) [inline release_notes/ocp-4-14-release-notes.adoc:379]:
```suggestion
The previously listed CAs stored in machine configuration files are removed, and the templated files found during cluster installation no longer exist.
```

Comment (skrthomas) [reply release_notes/ocp-4-14-release-notes.adoc:379]: IBMSG adivises caution when using above and suggests using previously or preceding instead. 
https://www.ibm.com/docs/en/ibm-style?topic=word-usage#a

Also, not sure if this makes sense here, but I was thinking active voice is better.

**Review by skrthomas — COMMENTED** [review]
A few small comments :)

File: release_notes/ocp-4-14-release-notes.adoc (hunk 4 above)
Comment (jldohmann) [reply release_notes/ocp-4-14-release-notes.adoc:379]: totally makes sense 🙂

File: release_notes/ocp-4-14-release-notes.adoc (hunk 3 above)
Comment (jldohmann) [reply release_notes/ocp-4-14-release-notes.adoc:369]: Ignition doesn't need backticks, I think you're right that controller config should either be lowercase or it should be in backticks e.g. `ControllerConfig` resource...this is a new area for me as well 😄

**Comment by rioliu-rh** [comment]
LGTM
/cc @sergiordlr @cdoern

File: release_notes/ocp-4-14-release-notes.adoc [hunk 5]
```diff
@@ -361,6 +361,25 @@ With this release, you can now provision an {product-title} cluster with Tang-en
...
 For more information, see xref:../installing/install_config/installing-customizing.adoc#installation-special-config-encryption-threshold_installing-customizing[Configuring an encryption threshold] and xref:../installing/install_config/installing-customizing.adoc#installation-special-config-storage-procedure_installing-customizing[Configuring disk encryption and mirroring].
 
+[id="ocp-4-14-new-cert-process"]
+==== Certificates are now handled by the Machine Config Daemon
+
+In previous {product-title} versions, the MCO read and handled certificates directly from machine configuration files. This led to rotation issues and created unwanted situations, like when certificates were stuck behind a paused machine config pool.
+
+With this release, certificates are no longer templated from bootstrap into machine configuration files. Instead, they are put directly into the Ignition object, written onto a disk using the controller config, and handled by the Machine Config Daemon (MCD) during regular cluster operation. The certs are then visible using the `ControllerConfig` resource.
+
+The MCD handles the following certificates:
```
[the remark this answers is not in the document]
Comment (cdoern) [reply release_notes/ocp-4-14-release-notes.adoc:371]: The Controller holds the certificate data but the MCD writes the data to disk so it is kind of a partnership. I guess we can be more specific here and say the Data is stored in and retrieved from the controller config and written to disk by the Daemon.

[the remark this answers is not in the document]
Comment (jldohmann) [reply release_notes/ocp-4-14-release-notes.adoc]: I've added a couple of sentences about the MCC and image registry certificates

**Pull request merged**